cobra/core/metabolite.py

```diff
@@ -206,7 +206,8 @@ def remove_from_model(self, destructive=False):
         """
         self._model.remove_metabolites(self, destructive)
 
-    def summary(self, threshold=0.01, fva=False, floatfmt='.3g', **kwargs):
+    def summary(self, solution=None, threshold=0.01, fva=False, floatfmt='.3g',
+                **kwargs):
```

Comment:
Suggest to remove the `**kwargs` here as only really relevant for the legacy solvers and those are not supported anyway